### ISSUE TITLE
Rails 6.1 gives errors when active record is enabled

### ISSUE THREAD

**Issue by johan-smits** [aws/aws-xray-sdk-ruby#56]
This happens because spec is removed.

```ruby
  Rails.application.config.xray = {
        name: "#{ENV['LC_LWS_ENV']}_#{Rails.application.class.name}",
        patch: %I[net_http aws_sdk],
        active_record: true,
        daemon_address: '127.0.0.1:2000',
        context_missing: 'LOG_ERROR'
    }
```

```
NoMethodError (undefined method `spec' for #<ActiveRecord::ConnectionAdapters::ConnectionPool:0x00007feeb1d0f708>
Did you mean?  inspect):

/var/app/current/vendor/bundle/ruby/2.7.0/gems/aws-xray-sdk-0.11.5/lib/aws-xray-sdk/facets/rails/active_record.rb:20:in `record'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/aws-xray-sdk-0.11.5/lib/aws-xray-sdk/facets/rails/active_record.rb:68:in `block in <top (required)>'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/notifications/fanout.rb:186:in `finish'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/notifications/fanout.rb:63:in `block in finish'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/notifications/fanout.rb:63:in `each'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/notifications/fanout.rb:63:in `finish'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/notifications/instrumenter.rb:45:in `finish_with_state'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/notifications/instrumenter.rb:30:in `instrument'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract_adapter.rb:688:in `log'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/postgresql_adapter.rb:691:in `exec_cache'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/postgresql_adapter.rb:657:in `execute_and_clear'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/postgresql/database_statements.rb:53:in `exec_query'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/database_statements.rb:536:in `select_prepared'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/database_statements.rb:67:in `select_all'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/query_cache.rb:101:in `block in select_all'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/query_cache.rb:118:in `block in cache_sql'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/concurrency/load_interlock_aware_monitor.rb:26:in `block (2 levels) in synchronize'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `handle_interrupt'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `block in synchronize'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `handle_interrupt'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activesupport-6.1.0/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `synchronize'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/query_cache.rb:109:in `cache_sql'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/connection_adapters/abstract/query_cache.rb:101:in `select_all'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/querying.rb:47:in `find_by_sql'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/statement_cache.rb:150:in `execute'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/core.rb:359:in `find_by'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/dynamic_matchers.rb:66:in `find_by_token'
/var/app/current/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.0/lib/active_record/dynamic_matchers.rb:20:in `method_missing'
```

**Comment by emish89** [comment]
You can do something like this:
`gem 'aws-xray-sdk', github: 'aws/aws-xray-sdk-ruby', ref: 'e6bbe20', require: ['aws-xray-sdk/facets/rails/railtie']`

Because with this commit is working.
Hoping to have this commit released as soon as possibile :)

**Comment by sisisin** [comment]
Hi, thank you for support Rails 6.1!

I'm using aws-xray-sdk-ruby with Rails 6.1 and ActiveRecord is enabled.
But, xray-sdk will give the following deprecation warning when run my application.

```
DEPRECATION WARNING: DatabaseConfig#config will be removed in 6.2.0 in favor of DatabaseConfigurations#configuration_hash which returns a hash with symbol keys (called from record at /Users/****/.asdf/installs/ruby/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/aws-xray-sdk-ruby-e6bbe200666b/lib/aws-xray-sdk/facets/rails/active_record.rb:24)
```

This warning caused by this PR's changes.
https://github.com/aws/aws-xray-sdk-ruby/pull/57/files#diff-d4f370d0fa2cb316a0ff51f25fa67093305353d0af7cbf061bcd6dc7c8a09baaR24

**Comment by seuros** [comment]
is this issue still current ? I see that master has support for 6.1 ?

**Comment by sisisin** [comment]
This issue resolved when #58 merged, so I think it should be closed

**Comment by johan-smits** [comment]
@willarmiros here all good.
